en/yohane.md

```diff
@@ -46,29 +46,29 @@ Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<User
 - `!alert <message>` - Send a notification to every user connected to bancho
 - `!alertuser  <username> <message>` - Send a notification to a specific user
-- `!whitelist <username> <mode> <type> [procedure]` - Melakukan whitelisting pada PP limit pada suatu user.
+- `!whitelist <username> <mode> <type> [procedure]` - Adds user to the PP limit whitelist.
     
-    | Parameter | Nilai | Fungsi |
+    | Parameter | Value | Function |
     | :-- | :-: | :-- |
-    | `mode` | `vanilla`, `relax` | Menentukan mode permainan yang di-*whitelist* |
-    | `type` | `total`, `all` | Menentukan jenis batasan yang di-*whitelist* |
-    | `procedure` | `add`, `set` | *Optional*<br>Menentukan operasi pengubahan nilai *whitelist* PP |
-- `!map <command> <type> <ID>` - Melakukan operasi pada suatu map(set)
+    | `mode` | `vanilla`, `relax` | Specifies the game mode for the *whitelist* |
+    | `type` | `total`, `all` | Specifies the type of limit for the *whitelist* |
+    | `procedure` | `add`, `set` | *Optional*<br>Specifies the operation of changing the PP value for the *whitelist* |
+- `!map <command> <type> <ID>` - Performs operations on a map(set)
     
-    | Parameter | Nilai | Fungsi |
+    | Parameter | Value | Function |
     | :-- | :-: | :-- |
-    | `command` | `rank`, `love`, `unrank`, `reset` | Melakukan pengubahan status map langsung. |
-    | `command`\* | `auto-rank`, `auto-love`, `auto-cancel` | **Khusus yang dapat mengatur autorank**<br>Menentukan kelayakan suatu map untuk mendapatkan auto-rank. |
-    | `type` | `set`, `set-of`, `map` | `set` - menggunakan **mapset ID** dalam mengambil data.<br>`set-of` - menggunakan **map ID** untuk mendapatkan **daftar map dengan mapset ID** dari **map ID** yang diberikan<br>`map` - menggunakan **map ID** dalam mengambil data. |
+    | `command` | `rank`, `love`, `unrank`, `reset` | Directly changes the status of the map. |
+    | `command`\* | `auto-rank`, `auto-love`, `auto-cancel` | **Can only be set by someone permitted to manage autoranks**<br>Sets the eligibility of a map to get auto-rank. |
+    | `type` | `set`, `set-of`, `map` | `set` - uses **mapset ID** to retrieve data.<br>`set-of` - uses **map ID** to get **a list of maps with the mapset ID** from the given **map ID**.<br>`map` - uses **map ID** to retrive data. |
     
-    Untuk lebih jelasnya, dapat memperhatikan [contoh berikut](https://cdn.discordapp.com/attachments/265909019976138754/801471868699410473/unknown.png)
+    For more details, you may take a look at [this example](https://cdn.discordapp.com/attachments/265909019976138754/801471868699410473/unknown.png).
 
 ### Note
-Jika kalian ingin menggunakan command melalui private message Yohane, tidak perlu menggunakan prefiks-"!"
-- Diluar itu, prefiks-"!" diperlukan untuk memanggil command Yohane.
+If you want to use commands through Yohane's private message, you don't need to use the "!"-prefix.
```

Comment:
```suggestion
If you're using commands through Yohane's private message, you can type your command without the "!"-prefix.
```

Reply:
word doubling with "command", I'd only change the first part

Reply:
could remove the latter